CONTRIBUTNG.md

```diff
@@ -0,0 +1,10 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
```

Comment:
"credit will always be given" this is the same formulation as the other PR, just curious, are you planning an auto generated list of committer page or something?

Reply:
It's just a usual sentence for a contributing page. A simple authors file could be added to give credit. For example : https://github.com/Ouranosinc/Magpie/blob/master/AUTHORS.rst